stl_test.go

```diff
@@ -37,3 +47,39 @@ func TestSTL(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, string(c), w.String())
 }
+
+func TestOPNSTL(t *testing.T) {
+	// Init
+	creationDate, _ := time.Parse("060102", "200110")
+	revisionDate, _ := time.Parse("060102", "200110")
+
+	// Open
+	s, err := astisub.OpenFile("./testdata/example-opn-in.stl")
+	assert.NoError(t, err)
+	//assertSubtitleItems(t, s)
```

Comment:
Why is this commented?

Reply:
I did not implement this part of the test. The new subtitle test file that I used has different data from the existing test files. I will remove the comment.